.php-cs-fixer.dist.php

```diff
@@ -45,6 +45,7 @@
         'general_phpdoc_annotation_remove' => ['annotations' => ['expectedDeprecation']], // one should use PHPUnit built-in method instead
         'header_comment' => ['header' => $header],
         'modernize_strpos' => true, // needs PHP 8+ or polyfill
+        'declare_equal_normalize' => ['space' => 'single'],
```

Comment:
I think we shouldn't have the spaces https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-12-extended-coding-style-guide-meta.md#441-php-fig-representative-results

Reply:
yes, you are right: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6201#discussion_r772412389